src/passes/RemoveUnusedTypes.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 WebAssembly Community Group participants
```

Comment:
```suggestion
 * Copyright 2022 WebAssembly Community Group participants
```
Almost 2023 :exploding_head: